src/com/serotonin/mango/rt/maint/work/EmailWorkItem.java

```diff
@@ -59,14 +61,20 @@ public static void queueEmail(String[] toAddrs, MangoEmailContent content, Runna
         queueEmail(toAddrs, content.getSubject(), content, postSendExecution);
     }
 
-    public static void queueEmail(String[] toAddrs, String subject, EmailContent content, Runnable[] postSendExecution)
-            throws AddressException {
+    public static void queueEmail(String[] toAddrs, String subject, EmailContent content, Runnable[] postSendExecution) {
         EmailWorkItem item = new EmailWorkItem();
 
-        item.toAddresses = new InternetAddress[toAddrs.length];
-        for (int i = 0; i < toAddrs.length; i++)
-            item.toAddresses[i] = new InternetAddress(toAddrs[i]);
+        Set<InternetAddress> addresses = new HashSet<>();
+        for (int i = 0; i < toAddrs.length; i++) {
+            try {
+                InternetAddress internetAddress = new InternetAddress(toAddrs[i]);
+                addresses.add(internetAddress);
```

Comment:
This try/catch looks great and now everything is working! 
If I were writing that highlighted fragment I wouldn't use temporary variable, but its not a problem :)